esque/topic.py

```diff
@@ -14,6 +13,24 @@
     unpack_confluent_config,
 )
 
+class TopicConfig:
```

Comment:
I think you could merge this class and Topic into one. 
Workflow would be like this: Create new `Topic` instance -> Pass Instance to `TopicController` -> Controller creates Topic and enriches instance with more information.

Reply:
But I don't have a cluster object in the command methods. I was thinking about two different constructors for the Topic class, but this is obviously not possible in Python.

Reply:
It's a bit hidden but you do. If you use `@pass_state` the state object is passed to your command which lazily creates a cluster object: https://github.com/real-digital/esque/blob/master/esque/cli/options.py#L31